handlers.go

```diff
@@ -115,23 +114,27 @@ func (h *orgsHandler) reloadOrgs(writer http.ResponseWriter, req *http.Request)
 	writeJSONMessageWithStatus(writer, "Reloading V1 organisations", http.StatusAccepted)
 }
 
-func (h *orgsHandler) HealthCheck() v1a.Check {
-	return v1a.Check{
+func (h *orgsHandler) HealthCheck() fthealth.Check {
+	return fthealth.Check{
 		BusinessImpact:   "Unable to respond to requests",
-		Name:             "Check service has finished initilising.",
+		Name:             "Check service has finished initialising.",
 		PanicGuide:       "https://sites.google.com/a/ft.com/ft-technology-service-transition/home/run-book-library/v1-people-transformer",
 		Severity:         1,
 		TechnicalSummary: "Cannot serve any content as data not loaded.",
 		Checker: func() (string, error) {
 			if h.service.isInitialised() {
 				return "Service is up and running", nil
 			}
-			return "Error as service initilising", errors.New("Service is initilising.")
+			return "Error as service initilising", errors.New("Service is initialising")
 		},
 	}
 }
 
-func (h *orgsHandler) getGTG() gtg.Status {
+func (h *orgsHandler) GTG() gtg.Status {
+	return gtg.FailFastParallelCheck([]gtg.StatusChecker{h.gtgCheck})()
```

Comment:
I think it's not necessary a parallel check for only one check, but we could live with that